packages/front-end/components/Features/SDKConnections/SDKConnectionsList.tsx

```diff
@@ -292,7 +292,18 @@ export default function SDKConnectionsList() {
             </div>
           ) : null}
         </>
-      ) : null}
+      ) : (
+        <>
+          {connections.length === 0 ? (
```

Comment:
I think that it would make more sense to invert the conditionals:
```typescript
      {connections.length === 0 ? (
        <>
          {canCreateSDKConnections ? (
            <div className="appbox p-5 text-center">
              <p>
                <strong>SDK Connections</strong> make it easy to integrate
                GrowthBook into your front-end, back-end, or mobile application.
              </p>
              <button
                className="btn btn-primary"
                onClick={(e) => {
                  e.preventDefault();
                  setModalOpen(true);
                }}
              >
                <GBAddCircle /> Create New SDK Connection
              </button>
            </div>
          ) : (<div className="appbox p-5 text-center">
              <p>
                You do not have permission to create SDK connections. Please
                contact your account administrator
              </p>
            </div>
          )
        </>) : null}
```